src/Pim/Bundle/CatalogBundle/Doctrine/ArrayApcCache.php

```diff
@@ -1,82 +0,0 @@
-<?php
-
-namespace Pim\Bundle\CatalogBundle\Doctrine;
-
-use Doctrine\Common\Cache\ApcuCache;
-use Doctrine\Common\Cache\ArrayCache;
-use Doctrine\Common\Cache\Cache;
-
-/**
- * Proxy for ApcuCache with a fall back on ArrayCache when using cli mode.
- *
- * In http mode, apc is enabled by default.
- *
- * In command line mode, apc is disabled by default, we advise to enable it with the option apc.enable_cli=1.
- *
- * This class replaces Doctrine\Common\Cache\ApcCache in the configuration, defined in doctrine/doctrine-bundle with
- * the parameter "doctrine.orm.cache.apc.class"
- *
- * @author    Antoine Guigan <antoine@akeneo.com>
- * @copyright 2013 Akeneo SAS (http://www.akeneo.com)
- * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
- */
-class ArrayApcCache implements Cache
```

Comment:
no more need of this class? means with doctrine 2.5 we have cache enabled even for cli calls?